### PR TITLE
Front#1

### PR DESCRIPTION
**Pull request by DanilFetter**

### CODE REVIEW

File: frontend/src/app/task-area/task-area.component.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+              id="taskDone"
+              name="happy"
+              value="yes"
+            />
+            <label for="taskDone"></label>
+            <span class="task-text">{{item.text}}</span>
+            <div class="task-tab">{{item.tabGroupe}}</div>
+            <div class="gaper"></div>
+            <img
+              class="dalaete-button"
```
Comment (aslanforyou) [inline frontend/src/app/task-area/task-area.component.html]: wrong spelling - 'delete-'

File: frontend/src/app/task-area/task-area.component.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+              [checked]="item.status"
+              class="custom-checkbox"
+              id="taskDone"
+              name="happy"
+              value="yes"
+            />
+            <label for="taskDone"></label>
+            <span class="task-text">{{item.text}}</span>
+            <div class="task-tab">{{item.tabGroupe}}</div>
+            <div class="gaper"></div>
```
Comment (aslanforyou) [inline frontend/src/app/task-area/task-area.component.html:28]: not needed to place dummy element to align delete button to right. Use flex alignment

File: frontend/src/app/task-area/task-area.component.scss [hunk 3]
```diff
@@ -0,0 +1,238 @@
+.wrapper {
```
Comment (aslanforyou) [inline frontend/src/app/task-area/task-area.component.scss:10]: This file is SASS but does not use it key feature as it should be (nesting elements)

File: frontend/src/app/task-area/task-area.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { TaskLoaderService, TaskInterface } from '../task-loader.service';
+@Component({
+  selector: 'app-task-area',
+  templateUrl: './task-area.component.html',
+  styleUrls: ['./task-area.component.scss'],
+})
+
+export class TaskAreaComponent implements OnInit {
+  displayedTask: TaskInterface[] = [];
```
Comment (aslanforyou) [inline frontend/src/app/task-area/task-area.component.ts]: better displayedTasks <- everybody will understand that it is an array/list (not a single item)

File: frontend/src/app/task-area/task-area.component.scss [hunk 5]
```diff
@@ -0,0 +1,238 @@
...
+  line-height: 14px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: #FFFFFF;
+}
+
+
+
+@media (max-width:1058px) {
```
Comment (aslanforyou) [inline frontend/src/app/task-area/task-area.component.scss:185]: this 1058px and 550px widths - what is the reference to use exactly this values? 
For example Bootstrap has another (as I remember)

File: frontend/src/app/task-area/task-area.component.ts (hunk 4 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.ts]: fixed

File: frontend/src/app/task-area/task-area.component.html (hunk 2 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html:28]: fixed

File: frontend/src/app/task-area/task-area.component.html (hunk 1 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.html]: fixed

File: frontend/src/styles.scss [hunk 6]
```diff
@@ -1 +1,7 @@
-/* You can add global styles to this file, and also import other style files */
+body, html, h1,h2,h3,p,span,button {
```
Comment (kleputin) [inline frontend/src/styles.scss]: Usually, there should be only 
`html, body {
  height: 100vh;
  width: 100%;
  margin: 0;
}`

File: frontend/src/styles.scss [hunk 7]
```diff
@@ -1 +1,7 @@
...
+  margin:0;
+}
+
+
+html, body { height: 100%; }
+body { margin: 0; font-family: 'Lato', sans-serif; }
```
Comment (kleputin) [inline frontend/src/styles.scss]: Isn't `margin` for body already settled in selector above?

File: frontend/src/assets/Vector.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<svg width="16" height="18" viewBox="0 0 16 18" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (kleputin) [inline frontend/src/assets/Vector.svg]: What is `Vector.svg`? Please, set more descriptive name.

Comment (kleputin) [reply frontend/src/assets/Vector.svg]: Also, can you use `svg` icon instead `png`, please? Just wondering, do you know what differences between `png` and `svg` icons?

File: frontend/src/app/task-loader.service.ts [hunk 9]
```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+export interface TaskInterface {
```
Comment (kleputin) [inline frontend/src/app/task-loader.service.ts:3]: Please, add line between interface and imports and add line between interface and decorator. Also, there should be not space between class and decorator.

File: frontend/src/app/task-area/task-area.component.ts [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+  constructor(private readonly taskLoaderService: TaskLoaderService) {}
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.ts]: Keep space between functions

File: frontend/src/app/task-area/task-area.component.ts [hunk 11]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+import { TaskLoaderService, TaskInterface } from '../task-loader.service';
+@Component({
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.ts:3]: Also, keep space between imports and decorators

File: frontend/src/app/task-area/task-area.component.ts [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-task-area',
+  templateUrl: './task-area.component.html',
+  styleUrls: ['./task-area.component.scss'],
+})
+
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.ts]: There should be not space between decorator and class

File: frontend/src/app/task-area/task-area.component.html [hunk 13]
```diff
@@ -0,0 +1,43 @@
...
+      <button class="btn">College</button>
+      <button class="btn">Payments</button>
+    </div>
+    <div class="task-area">
+      <h1>All Tasks</h1>
+      <input
+        class="new-input"
+        type="text"
+        placeholder="Add a new task insdie ‘All’ category"
+        name=""
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: Why is this needed?

File: frontend/src/app/task-area/task-area.component.html [hunk 14]
```diff
@@ -0,0 +1,43 @@
...
+        id=""
+      />
+      <div class="task-block">
+        <ul class="task-list">
+          <li class="task" *ngFor="let item of displayedTask">
+            <input
+              type="checkbox"
+              [checked]="item.status"
+              class="custom-checkbox"
+              id="taskDone"
```
Comment (kleputin) [inline frontend/src/app/task-area/task-area.component.html]: The `id` is not unique

**Review by kleputin — CHANGES_REQUESTED** [review]
Please, address comments and push changes.

File: frontend/src/app/task-area/task-area.component.ts (hunk 12 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.ts]: Fixed

File: frontend/src/app/task-area/task-area.component.ts (hunk 11 above)
Comment (DanilFetter) [reply frontend/src/app/task-area/task-area.component.ts:3]: fixed

File: frontend/src/styles.scss (hunk 7 above)
Comment (DanilFetter) [reply frontend/src/styles.scss]: fixed
